### ISSUE TITLE
Provide data binding to `url` of labels

### ISSUE THREAD

**Issue by notacouch** [amcharts/amcharts4#468]
Most string properties have access to data placeholders, so there's a reasonable expectation for the following to work, especially given [the documentation suggests the parent's data would be checked](https://www.amcharts.com/docs/v4/concepts/formatters/formatting-strings/#Data_source):
```javascript
bullet1.label.url = "https://www.google.com/?q={name}"
```
But it just gets interpreted literally.

Instead, even for simple adjustments, users have to use adapters:

```javascript
// Goal:
// bullet1.label.url = "https://www.google.com/?q={name}"
// instead...
// If we want to utilize label data or properties, we can
// do so through adapters on the url property itself.
bullet1.label.adapter.add("url", function(url, label) {
  var query = "";
  var data = label.dataItem;
  if (data.dataContext && data.dataContext.name) {
    query = "?q=" + data.dataContext.name;
  }
  return url + query;
});
```
As you can see there, the label itself already has populated dataItems, no need to go further up the chain.

[Sample on CodePen](https://codepen.io/team/amcharts/pen/e13b83f995e15f885c74c64a5d63e568?editors=0010) (line 439)

**Comment by martynasma** [comment]
Probably also need to think how to implement urlncoding of injected value.

**Comment by martynasma** [comment]
Implemented. Will be released in next version. (later today)

Also urlencoding via inline function:

```
series.columns.template.url = "https://www.google.com/search?q={category.urlEncode()}";
```

**Comment by martynasma** [comment]
Fixed/implemented in [4.0.0-beta.61](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.61).

## [4.0.0-beta.61] - 2018-10-19

### Changed
- If element has `url` set, when clicked, it will now parse for data placeholders in curcly brackets, e.g. `"{category}"` allowing have dynamic data-driven URLs. Note, the elements are not url-encoded so you need to specify encoding manually, e.g.: `series.columns.template.url = "https://www.google.com/search?q={category.urlEncode()}";`.

### Fixed
- `Label.fullWords = false` was not working properly.
- `AxisLabelCircular` was positioning labels incorrectly, if label.radius was negative (since 59).
- `"grid"` layout in `Container` was incorrectly layouting elements if their `width` was set in percent.
- [IE] Multi-line truncated labels were not working properly on IE.
- [IE] was incorrectly zIndexing elemens (since version 56).
- [IE] tooltip initial flicker solved.
- [IE] was incorrectly sorting multiple series tooltips (if `Cursor` was used).

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
